### PR TITLE
Added fee tile for send scene.

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: src/components/themed/AddressTile.js [hunk 1]
```diff
@@ -184,44 +208,30 @@ class AddressTileComponent extends React.PureComponent<Props, State> {
 
       // set address
       onChangeAddress(parsedUri.publicAddress, { fioAddress })
+      this.setState({ enterAddress: false, resolvingAddress: false })
```
Comment (andreyvEze) [inline src/components/themed/AddressTile.js:211]: move resolvingAddress before conditions

**Comment by thehobbit85** [comment]
@paullinator I would like you to approve/comment on this PR since there is UX change (the close button).

**Comment by paullinator** [comment]
Things look fine on the visuals, but the screen is about to get refactored by @amanalili  and most of the styling changes will be complete. Any other changes from this PR should be re-implemented on top of @amanalili 's changes

**Comment by thehobbit85** [comment]
@paullinator Approved but waiting till next release for you to merge.

**Pull request merged**